src/util/resize.js

```diff
@@ -0,0 +1,50 @@
+export const DIVIDER_HEIGHT = 4;
+export const DEFAULT_VERTICAL_FLEX = [
+  `1 1 ${DIVIDER_HEIGHT}px`,
+  `1 1 ${DIVIDER_HEIGHT}px`,
```

Comment:
There may be some subtlety I’m missing here with the relationship of `<Editor>`, `<EditorContainer>`, and the divider though…